src/store/config.ts

```diff
@@ -24,8 +24,8 @@ class ConfigStoreImpl implements ConfigStore {
         this.storeData = {} as AppConfigStore;
     }
 
-    storeConfigInfo(store: AppConfigStore): Promise<void> {
-        return this.ppClient.kvSet(StoreKeys.config, store);
+    async storeConfigInfo(store: AppConfigStore): Promise<void> {
+        await this.ppClient.kvSet(StoreKeys.config, store);
```

Comment:
Returning and awaiting will do the same thing here, it's just that returning is cleaner imo. Since `kvSet` is already returning a promise, returning its value will also return a promise, not requiring this function to be declared as `async`. It's a matter of preference though since we are returning `Promise<void>`.

Reply:
I can see both ways, since `storeConfigInfo ` is basically a wrapper around the `kvset` one could indeed return the promise and let the upper-level calls await on it - or, instead - await all the way to the top (with the function that returns the promise at the bottom of the chain - the only one).